src/main/java/mate/academy/springboot/datajpa/services/ProductService.java

```diff
@@ -0,0 +1,20 @@
+package mate.academy.springboot.datajpa.services;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+import mate.academy.springboot.datajpa.models.Product;
+
+public interface ProductService {
+    Product save(Product product);
+
+    Product findById(Long id);
+
+    void delete(Long id);
+
+    Product update(Product product);
+
+    List<Product> findAllByPricePriceBetween(BigDecimal from, BigDecimal to);

```

Comment:
Method name 'findAllByPricePriceBetween' could be more descriptive, consider renaming it to 'findAllByPriceBetween'.

Reply:
It was my bad). It was typo.